README.md

```diff
@@ -100,3 +100,30 @@ Please refer to the [CONTRIBUTING.md](CONTRIBUTING.md) file for more information
 
 
 [helm]: https://helm.sh/
+
+# Deploy RHTAP
+Follow the below steps to deploy RHTAP on Openshift cluster.
+
+1.Clone the repositry.Run the command 'make` from the rhtap-cli directory.
+This will create a bin directory.
+
+2.Edit the config.yaml file for select or deselect the components from installation.
+
+  Eg : Change the lines as below to disable installation of components ACS and Quay
+
+  `redHatAdvancedClusterSecurity:
+      enabled: false`
+      
+  `redHatQuay:
+      enabled: false`
+      
+3.Run the command `bin/rhtap-cli` to display help text that shows all the supported commands and options
+
+4.Run the command `bin/rhtap-cli integration` to provide integrations to external components
+  
+  `bin/rhtap-cli integration --help` will list the options supported
```

Comment:
Another place where it should become a code block, but this time Bash formatted.

Reply:
Done with all changes